### PR TITLE
feat: add polygon zkevm mainnet for clober

### PR DESCRIPTION
**Pull request by graykode**
**NOTE**

We supports new chain: Polygon zkevm
https://github.com/DefiLlama/DefiLlama-Adapters/pull/5337

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. The protocol is usually listed within 24 hours of merging the PR
4. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
5. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
6. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
7. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---

### CODE REVIEW

**Comment by llamatester** [comment]
Error while running adapter at projects/clober: 
        
 
  
    
    
    Error: Multicall failed!
    Truncated error stack:
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@defillama/sdk/build/abi/index.js:232:35

**Comment by llamatester** [comment]
The adapter at projects/clober exports TVL: 
        
 
 
    ethereum                  523.90 k
    polygon                   231.82419661119752
    polygon_zkevm             5.41 k
    arbitrum                  29.52 k
    
    total                    559.05 k

File: projects/clober/index.js [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
 
-const FACTORY = "0x93A43391978BFC0bc708d5f55b0Abe7A9ede1B91"
-
 async function tvl(_, _b, _cb, { api }) {
-    let tokenAddresses = await api.fetchList({  lengthAbi: abi.nonce, itemAbi: abi.computeTokenAddress, target: FACTORY}) 
-    tokenAddresses = tokenAddresses.flat()
-    const markets = await api.multiCall({  abi: abi.market, calls: tokenAddresses })
-    const base = await api.multiCall({  abi: abi.baseToken, calls: markets}) 
+    const chainId = await api.getChainId()
+    const markets = (await fetchURL(`https://prod.clober-api.com/${chainId}/markets`)).data.markets.map((market) => market.address)
```
Comment (g1nt0ki) [inline projects/clober/index.js:7]: I am not a fan of relying on external apis as they to break in the future, is there some alternative for this? like fetching via event logs?

Comment (graykode) [reply projects/clober/index.js]: @g1nt0ki https://github.com/DefiLlama/DefiLlama-Adapters/pull/6194/commits/6a3848cd1f5e712dfdfefb9f9fa04f7aa2cbef66

Good! I've fixed it!

**Comment by llamatester** [comment]
The adapter at projects/clober exports TVL: 
        
 
 
    ethereum                  523.01 k
    polygon                   224.27027213151837
    arbitrum                  27.94 k
    polygon_zkevm             4.39 k
    
    total                    555.56 k

**Comment by g1nt0ki** [comment]
@graykode thanks for fixing it

**Pull request merged**